packages/shared/graphql/clientIdHeader.ts

```diff
@@ -0,0 +1,14 @@
+import { generateUUID } from "devtools/shared/generate-uuid";
+
+// Generate a client ID so that we can easily filter backend requests down to a specific client
+// browser tab. It might seem like we should just be attaching the session ID here, but devtools
+// makes requests before the session is established, so we use an arbitrary ID instead.
+export const GRAPHQL_CLIENT_ID = generateUUID();
+// This constant is also referenced in our GraphQL code.
```

Comment:
Is this comment referring to code that runs on the backend?

Reply:
oh yes, sorry let me make that clearer.